plugins/performance-lab/includes/site-health/far-future-headers/hooks.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Hook callbacks used for far-future headers.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
```

Comment:
Per https://github.com/WordPress/performance/pull/1815:

```suggestion
// @codeCoverageIgnoreStart
if ( ! defined( 'ABSPATH' ) ) {
	exit; // Exit if accessed directly.
}
// @codeCoverageIgnoreEnd
```